test/syntax.jl

```diff
@@ -1436,3 +1436,6 @@ let ex = Meta.parse("@test27521(2) do y; y; end")
                      fex)
     @test macroexpand(@__MODULE__, ex) == Expr(:tuple, fex, 2)
 end
+
+# pr #27668
+@test (try Meta.parse("map(1) do x...; end"); catch; 0; end) isa Expr
```

Comment:
There's no need for the try-catch, since an unexpected error will cause the test to fail anyway.

Reply:
If it throw, will the exception block the tests after it?

Reply:
No; `@test` wraps its argument in try-catch and records whether an exception was thrown.